webui/react/src/stores/tasks.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { createContext, PropsWithChildren, useCallback, useContext, useState } from 'react';
+
+import { getActiveTasks } from 'services/api';
+import { TaskCounts } from 'types';
+import handleError from 'utils/error';
+
+type TasksContext = {
+  activeTasks: TaskCounts;
+  updateActiveTasks: (fn: (ws: TaskCounts) => TaskCounts) => void;
+};
+
+const TasksContext = createContext<TasksContext | null>(null);
+
+export const TasksProvider: React.FC<PropsWithChildren> = ({ children }) => {
+  const [state, setState] = useState<TaskCounts>({
```

Comment:
Instead of returning a fake response, it's better to return `NotLoaded` to callers of `useActiveTasks` so they can handle the loading state.

Reply:
✅ ok I have Loadable integrated back into this and can init this as NotLoaded